internal/pkg/app/testdata/curl.sh

```diff
@@ -14,104 +14,104 @@
 
 echo "dev"
 
-curl -X PUT https://dev-us-west-2.gears.comcast.com/ears/v1/orgs/myorg/applications/myapp/config --data-binary @quota.json | jq .
+curl -X PUT https://dev-us-west-2.gears.comcast.com/ears/v1/orgs/comcast/applications/myapp/config --data-binary @quota.json | jq .
```

Comment:
I thought we don't want comcast specific words?

Reply:
if i don't change this to "comcast" i cannot test this with any of our sat tokens - i felt like the string "comcast" doesn't give too much information away :)